### PR TITLE
Adding download form for slides in UNS webinar

### PR DESCRIPTION
**Pull request by Hasmin-AC**
## Checklist

 - [ ] I have read the [contribution guidelines](https://github.com/FlowFuse/flowfuse/blob/main/CONTRIBUTING.md)
 - [ ] I have considered the performance impact of these changes
 - [ ] Suitable unit/system level tests have been added and they pass
 - [ ] Documentation has been updated

### CODE REVIEW

File: src/webinars/2024/building-unified-namespace-using-nodered-mqtt.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
+subtitle: Grey Dziuba, OT Data and Community Strategist, and Kudzai Manditereza, Developer Advocate at HiveMQ, discuss MQTT and Node-RED and how it pertains to the Unified Namespace, UNS.
 image: /images/webinars/building-unified-namespace-uns-using-node-red-and-mqtt-webinar-2024-march.jpg
 date: 2024-03-26
 time: 17:00 CET (11:00am ET) 
 duration: 60
 video: U4njRFVLTjI
 hosts: ["grey-dziuba"]
 hubspot:
-    formId: b4ec9aa1-8648-47b3-b908-ec8597e8735d
+    formId: 49e2f5b2-2e8b-4be0-808c-57c1bd691224
```
Comment (Hasmin-AC) [inline src/webinars/2024/building-unified-namespace-using-nodered-mqtt.md:11]:
```suggestion
    
    formId: b4ec9aa1-8648-47b3-b908-ec8597e8735d
    downloadFormId: 49e2f5b2-2e8b-4be0-808c-57c1bd691224
```

**Comment by Hasmin-AC** [comment]
@Yndira-E I'm not sure why the Register title is still showing on this page. Can you please help remove it?

**Comment by Hasmin-AC** [comment]
@Yndira-E The register title is still showing and I'm not sure how to remove it. Can you please help?

**Comment by Yndira-E** [comment]
@Hasmin-AC 

> The register title is still showing

If you mean that it's still showing it as an upcoming webinar, it's because it only goes to the past webinars list the day after the event, it's automated, no need to do anything. The preview now it's outdated because it was built yesterday, if we merge now, it should show it in the past webinars list.

**Review by ZJvandeWeg — APPROVED** [review]

**Pull request merged**